src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfSendungOverviewJsonDeserializerTest.java

```diff
@@ -62,7 +63,7 @@ public SrfSendungOverviewJsonDeserializerTest(String aJsonFile, Optional<String>
   public void test() {
     JsonElement jsonElement = JsonFileReader.readJson(jsonFile);
     
-    Optional<SrfSendungOverviewDTO> actual = target.deserialize(jsonElement, SrfSendungOverviewDTO.class, null);
+    Optional<SendungOverviewDto> actual = target.deserialize(jsonElement, SendungOverviewDto.class, null);
     
     assertThat(actual.isPresent(), equalTo(true));
     assertThat(actual.get().getNextPageId(), equalTo(expectedNextPageId));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "actual.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)